app/src/test/java/com/x8bit/bitwarden/data/auth/datasource/network/service/IdentityServiceTest.kt

```diff
@@ -327,9 +329,97 @@ class IdentityServiceTest : BaseServiceTest() {
         assertTrue(result.isFailure)
     }
 
+    @Test
+    fun `registerFinish success json should be Success`() = runTest {
+        val json = """
+            {
+              "captchaBypassToken": "mock_token"
+            }
+            """
```

Comment:
We have usually made these constants at the bottom of the file.

Makes the actual test read easier and opens it up for reusability in the future.

Reply:
Moved it and changed other places where they were used